src/ploigos_step_runner/step_implementers/create_container_image/buildah.py

```diff
@@ -100,6 +102,35 @@ def _required_config_or_result_keys():
         """
         return REQUIRED_CONFIG_OR_PREVIOUS_STEP_RESULT_ARTIFACT_KEYS
 
+    def get_file_hash(self, application_name, service_name, tar_file):
```

Comment:
@blakerollins @jthompkins this function still needs to be updated to use the better way we found to get the digest.
`buildah --storage-driver=vfs images --digests --format "{{.Digest}}" IMAGE_REF`

Reply:
https://github.com/ploigos/ploigos-step-runner/pull/146/commits/ecb88a44df022f75deed85c772e2f2d7fcc6e98f